platformio.ini

```diff
@@ -103,4 +105,42 @@ lib_deps =
   ${common.lib_deps_shared}
   arkhipenko/TaskScheduler@^3.8.5
   https://github.com/ssilverman/QNEthernet#v0.26.0
-  https://github.com/hytech-racing/shared_firmware_interfaces.git
\ No newline at end of file
+  https://github.com/hytech-racing/shared_firmware_interfaces.git
+
+
+[env:test_pedals_env]
+check_tool = clangtidy
+check_flags =
+  clangtidy: --config-file=./.clang-tidy
+check_severity = medium, high
+test_framework=googletest
+build_unflags = -std=gnu++11
+build_src_filter = 
+  +<*.cpp>
+  -<test_ethernet_main.cpp>
+  -<test_adcs.cpp>
+  -<main.cpp>
+build_flags = 
+  -std=c++17
+  -D TEENSY_OPT_SMALLEST_CODE
+check_src_filters =
+  +<include/*>
+  +<lib/*>
+  +<src/*>
+  -<src/old_main.cpp>
+
+platform = teensy
+board = teensy41
+framework = arduino
+monitor_speed = 115200
+upload_protocol = teensy-cli
+test_ignore =
+  test_interfaces
+  test_systems
+lib_deps =
+  ${common.lib_deps_shared}
+  arkhipenko/TaskScheduler@^3.8.5
+  https://github.com/ssilverman/QNEthernet#v0.26.0
+  https://github.com/hytech-racing/shared_firmware_interfaces.git
+  https://github.com/ssilverman/QNEthernet#v0.26.0
+  etlcpp/Embedded Template Library - Arduino@^20.38.2
```

Comment:
I don't think this should be necessary, since we're already including the etl library in common lib deps.